src/classes/User.js

```diff
@@ -55,6 +55,37 @@ class User {
     const filteredRecipe = this.repo.findRecipesByIngredient(ingredientName, this.favoriteRecipes);
     return filteredRecipe;
   }
+
+  returnPantryIngredients() {
+    let matchingIngredient = 'banana';
```

Comment:
Does banana live here now??

Reply:
This can be a `const` since the value is never re-assigned.